src/dbally/gradio/gradio_interface.py

```diff
@@ -28,6 +28,23 @@ async def create_gradio_interface(user_collection: Collection, preview_limit: in
     return gradio_interface
 
 
+def find_event_buffer() -> Optional[BufferEventHandler]:
+    """
+    Searches through global event handlers to find an instance of BufferEventHandler.
+
+    This function iterates over the list of global event handlers stored in `dbally.global_event_handlers`.
+    It checks the type of each handler, and if it finds one that is an instance of `BufferEventHandler`, it
+    returns that handler. If no such handler is found, the function returns `None`.
+
+    Returns:
+        The first instance of `BufferEventHandler` found in the list, or `None` if no such handler is found.
+    """
+    for handler in dbally.global_event_handlers:
+        if type(handler) is BufferEventHandler:  # pylint: disable=C0123
```

Comment:
Why not `isinstance(...)`?

Reply:
Due to inheritence 
```isinstance(handler, BufferEventHandler)``` would also return true for CLIEventHandler.
BufferEventHandler inherits from CLIEventHandler

Reply:
Are you sure? ;) As `BufferEventHandler` inherits from CLI handler (and not the other way around) isinstance won't return `True` in that case